src/components/Repos/Repos.module.css

```diff
@@ -0,0 +1,67 @@
+.wrap {
+  width: 100%;
+  margin-top: 5px;
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border: 2px solid dodgerblue;
+  border-radius: 8px;
+  box-sizing: border-box;
+  text-decoration: none;
+  transition: all .3s ease-in-out;
+}
+
+.wrap:hover{
+  background-color: dodgerblue;
+}
+
+.wrap:hover .name{
+  color: #fff;
+}
+
+.wrap:hover .desc{
+  color: #fff;
+}
+
+.name {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".name" to come before selector ".wrap:hover .name" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/alde_react/pullRequest?prid=5576104)